docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,104 @@
+<amplify-callout>
+When using the `@auth` directive with DataStore, currently only following fields are supported:
+
+1. `allow` with "owner" and "groups"
+2. `ownerField`
+3. `groups`
+4. `operations`
+
+</amplify-callout>
+
+## Updated schema
+
+Create a Model that is read-only.
```

Comment:
The following schema is an example of a model named `Todo` that is readable and writable by the creator, and read only to everyone else.

Reply:
```suggestion
The following schema is an example of a model named `Todo` that is readable and writable by the creator, and read only to everyone else.
```